sdk/storage/Azure.Storage.Files.Shares/src/ShareFileClient.cs

```diff
@@ -4583,6 +4584,7 @@ private void SetNameFieldsIfNull()
         /// A <see cref="RequestFailedException"/> will be thrown if
         /// a failure occurs.
         /// </remarks>
+        [ForwardsClientCalls]
```

Comment:
What this not working without `[ForwardsClientCalls]`?

Reply:
Yes tests failed diagnostic scope without this. Additionally, all our other overloads have it except these ones, as well as equivalent blobs/datalake methods, so we brought these up to speed.